scipy/stats/_hypotests_pythran.py

```diff
@@ -0,0 +1,49 @@
+#pythran export _Aij_pythran(float[:,:], int, int)
+#pythran export _Aij_pythran(int[:,:], int, int)
+def _Aij_pythran(A, i, j):
```

Comment:
So `_pythran` is already in the file name, hence I think it'd be easier not to have it in all function names too.